.github/workflows/deploy-devnet.yml

```diff
@@ -14,12 +14,22 @@ jobs:
       with:
         submodules: "recursive"
 
-    - name: Create Image Tarball for Node
+    - name: Set up Docker Buildx
+      uses: docker/setup-buildx-action@v1
+
+    - name: Login to Docker Hub
+      run: docker login -u atletanetwork -p ${{ secrets.REGISTRYPASS }} docker.io
+
+    - name: Build
       run: |
-        docker build -t sportchain-node .
-        docker save -o sportchain-node.tar sportchain-node
-        mkdir -p artifacts
-        mv sportchain-node.tar artifacts/
+          DATE=$(date +%d%m%y)
+          BUILD_COUNT=$(( ($(date +%s) / 60 / 60 / 24) % 10 ))
```

Comment:
I meant a situation when we deploy multiple times during a day. In that case date-based versioning wouldn't work so we need some increment to our version. by default, it's always "1", but in case of additional versions during a date, it may be "2", "3", etc. for this we need to take the last deployed version on the date, parse it and increment the number. it's readable, but I agree it might be complex to configure. otherwise consider adding a timestamp up to seconds instead. but it should be incremental by nature, it should be higher than the previous version.

Reply:
Fixed with added incrementation:
```
            SECONDS_SINCE_EPOCH=$(date +%s)
            SECONDS_IN_DAY=86400
            DAY_NUMBER=$((SECONDS_SINCE_EPOCH / SECONDS_IN_DAY))
            BUILD_COUNT=$(( DAY_NUMBER % 10 ))
            echo "::set-output name=build_count::$BUILD_COUNT"
```